components/chart/header/index.tsx

```diff
@@ -14,6 +84,13 @@ export default function Header({
           <span className="translate">Search Nodes</span>
         </button>
       </div>
+      <div className="leftSide">
+        <BCTagsInput
```

Comment:
Can you please try this package
https://www.npmjs.com/package/react-tag-input
I think it will work with us better than the old one, also it supports suggestions